salt/_modules/metalk8s_kubernetes_cronjob.py

```diff
@@ -0,0 +1,212 @@
+"""Utiliy methods to interact with Kubernetes Cron Job in MetalK8s.
+"""
+
+import time
+from salt.exceptions import CommandExecutionError
+
+__virtualname__ = "metalk8s_kubernetes_cronjob"
+
```

Comment:
For consistency let's define a simple `__virtual__` function

```suggestion

def __virtual__():
    return __virtualname__
```